services/static-webserver/client/source/class/osparc/desktop/credits/UsageTableModel.js

```diff
@@ -71,15 +50,30 @@ qx.Class.define("osparc.desktop.credits.UsageTableModel", {
     }
   },
 
+  statics: {
+    SERVER_MAX_LIMIT: 49,
+    COLUMN_ID_TO_DB_COLUMN_MAP: {
+      0: "root_parent_project_name",
+      1: "node_name",
+      2: "service_key",
+      3: "started_at",
+      // 4: DONT SORT BY DURATION
```

Comment:
Please check that below is not an error.

This looks suspicious. Why is there a `// 4:` also I would expect the index to also include a 4, you don't have one

Reply:
if you don't need 4 I would suggest something like
```suggestion
      // 4: (not used) SORTING BY DURATION
```